Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -1588,6 +1602,36 @@ def write_threat_note(client: Client, args: dict) -> CommandResults:
         raw_response=threat_notes)
 
 
+def get_threat_notes(client: Client, args: dict) -> CommandResults:
+    """
+    Get the note of a particular threat.
+    """
+    threat_id = args.get('threat_id')
+
+    context_entries = []
+    notes = client.get_threat_notes_request(threat_id)
+    if notes:
```

Comment:
What do we return if there are no notes found?

Reply:
If no notes are found, nothing is set in the context and the readable output is :
"
Sentinel One - Get Threat Notes
No entries.
"